### PR TITLE
First integration test

### PR DESCRIPTION
**Pull request by julien51**
This adds a directory at the root which will include the integration tests.
These integration tests *will* compile the smart contracts, deploy them on a local node (ganache)
and then run the react/next app and assert things.

### CODE REVIEW

**Comment by julien51** [comment]
Note that for now these are not run on PRs. I want to only do that when we're sure about how this works ;)

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Integration tests
+
+This is run to test integration tests which test Unlock as a whole.
+Other tests (unit tests for smart contracts or for react app) belong in their respective folders.
+
+This is useful to ensure that all blocks work "together" if they already work in isolation.
+We do not expect developers to necessarily run these tests for nost feature work but they will be
+run on every pull request as part of the CI/CD process.
+
+We use pupeteer.
```
Comment (benwerd) [inline tests/README.md]: (Typo: Puppeteer)

Comment (benwerd) [reply tests/README.md]:
```suggestion
We use Puppeteer.
```

File: tests/helpers/environment.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  async setup() {
+    await super.setup()
+  }
+
+  async teardown() {
+    await super.teardown()
+  }
+}
+
+module.exports = UnlockEnvironment
```
Comment (benwerd) [inline tests/helpers/environment.js]: Nit: EOF line break

**Pull request merged**